pyproject.toml

```
@@ -0,0 +1,29 @@
+[project]
+name = "rapyuta-io-sdk-v2"
+dynamic = ["version"]
+#version = "1.0.0"
+description = "Version 2 for Rapyuta.io SDK"
+dependencies = [
+    "httpx>=0.27.2",
+    "tenacity>=9.0.0",
+]
+readme = "README.md"
+requires-python = ">= 3.8"
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[tool.rye]
+managed = true
+dev-dependencies = []
+
+[tool.hatch.version]
+path = "rapyuta_io_sdk_v2/__init__.py"
+pattern = "^__version__ = ['\"](?P<version>.+)['\"]"
```

`pattern` is not required. 
```suggestion
[tool.hatch.version]
path = "rapyuta_io_sdk_v2/__init__.py"
```